guide/asynchronous-invocation/src/stock_quote_summary_service/tests/async_service_test.bal

```diff
@@ -1,14 +1,14 @@
 import ballerina/test;
 import ballerina/http;
 
-@test:Config
+@test:Config{}
 function testStockSummaryService() {
     // Invoking the main function
     http:Client httpEndpoint = new("http://localhost:9090");
 
-    json expectedPayload = { "GOOG": "Connection refused: localhost/127.0.0.1:9095",
-        "APPL": "Connection refused: localhost/127.0.0.1:9095", "MSFT":
-        "Connection refused: localhost/127.0.0.1:9095" };
+    json expectedPayload = { "GOOG": "Something wrong with the connection",
+        "APPL": "Something wrong with the connection", "MSFT":
+        "Something wrong with the connection" };
```

Comment:
```suggestion
        "Something is wrong with the connection" };
```

Reply:
These text in "test" files are just validating output form the remote service. Shall we leave the as is.

Reply:
+1